app/Commands/Sites/CreateCommand.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+namespace App\Commands\Sites;
+
+use App\Commands\BaseCommand;
+use App\Commands\Concerns\HasServerIdParameter;
+use App\Helpers\OptionsHelper;
+use App\Questions\Ask;
+use App\Questions\Choice;
+use App\Questions\Confirm;
+use App\Questions\HasQuestions;
+use Illuminate\Support\Str;
+
+class CreateCommand extends BaseCommand
+{
+    use HasQuestions;
+    use HasServerIdParameter;
+
+    protected $signature = 'sites:create
+                            {server_id? : Server ID}
+                            {--installation_method= : Type of installation (wp or blank)}
+                            {--domain= : Domain name}
+                            {--site_user=}
+                            {--db_name=}
+                            {--db_user=}
+                            {--db_pass=}
+                            {--wp_title=}
+                            {--wp_admin_user=}
+                            {--wp_admin_email=}
+                            {--wp_admin_pass=}
+                            {--php_version=}
+                            {--page_cache_enabled}
+                            {--https_enabled}
```

Comment:
We should probably be sticking to kebab-case for parameters to keep things matching with the parameters we inherit from the framework (e.g. --no-interaction). 

It looks like we already have a mixture of kebab-case vs. snake_case throughout the repo. Making sure we have a standard in mind and that this PR adheres to it would be nifty.

```suggestion
                            {server-id? : Server ID}
                            {--installation-method= : Type of installation (wp or blank)}
                            {--domain= : Domain name}
                            {--site-user=}
                            {--db-name=}
                            {--db-user=}
                            {--db-pass=}
                            {--wp-title=}
                            {--wp-admin-user=}
                            {--wp-admin-email=}
                            {--wp-admin-pass=}
                            {--php-version=}
                            {--page-cache-enabled}
                            {--https-enabled}
```

Reply:
Done